godtools/ViewControllers/Platform/LanguageSettingsViewController.swift

```diff
@@ -16,6 +16,12 @@ class LanguageSettingsViewController: BaseViewController {
     
     var delegate: LanguageSettingsViewControllerDelegate?
     
+    override var screenTitle: String {
+        get {
+            return "Language Settings"
```

Comment:
Let's make sure we don't forget to externalize strings. Can we start that now so we get most of them early on?